workspaces/js/css/vars.css

```diff
@@ -53,6 +53,10 @@
   --flows-modal-padding: var(--flows-size-large);
   --flows-modal-minWidth: 320px;
   --flows-modal-maxWidth: 640px;
+
+  --flows-banner-padding: var(--flows-size-large);
```

Comment:
Here I'd split it so that the inner and outer padding is separate.

You will most likely want to change where the banner is positioned on the screen but it shouldn't be tied to the inner padding of the content.